spec/suites/map/MapSpec.js

```diff
@@ -2016,6 +2016,37 @@ describe("Map", function () {
 		});
 	});
 
+	describe('#panInsideBounds', function () {
+
+		it("throws if map is not set before", function () {
+			expect(function () {
+				map.panInsideBounds();
+			}).to.throwError();
+		});
+
+		it("throws if passed invalid bounds", function () {
+			expect(function () {
+				map.panInsideBounds(0, 0);
+			}).to.throwError();
+		});
+
+		it("doesn't pan if already in bounds", function () {
+			map.setView([0, 0]);
+			var bounds = L.latLngBounds([[-1, -1], [1, 1]]);
+			var expectedCenter = L.latLng([0, 0]);
+			expect(map.panInsideBounds(bounds)).to.be(map);
+			expect(map.getCenter().distanceTo(expectedCenter)).to.be.lessThan(1);
```

Comment:
Please use latlng validation for the center
```suggestion
			expect(map.getCenter()).to.be.nearLatLng(expectedCenter);
```

Reply:
Thank you for checking my pull request. I made the changes you requested. I hope this looks better. Please let me know if there is anything else I need to modify.